core/domain/user_jobs_continuous.py

```diff
@@ -418,6 +471,27 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
+        """Implements the map function (generator).
+        Computes exploration data for every contributor and owner of the exploration.
+
+        Args:
+            item: The parameter passed to this function is a single element of
+            the type given by entity_class().
+
+        Returns:
+            This function may yield as many times as appropriate the key/value 2-tuples. 
```

Comment:
See above for suggestions on rewording. (If the only possible case is (id, exploration_data) then omit the "for example", since that phrasing suggests that other return types are possible).